backend/src/main/java/ai/verta/modeldb/config/DatabaseConfig.java

```diff
@@ -12,6 +14,7 @@
   public Integer connectionTimeout = ModelDBConstants.CONNECTION_TIMEOUT_DEFAULT;
 
   public RdbConfig RdbConfiguration;
+  public List<MigrationConfig> migrations;
```

Comment:
Migration format is like Map structure
```
migration:
  DATASET_VERSIONING_MIGRATION:
    enable: false
    record_update_limit: 100
  SUB_ENTITIES_OWNERS_RBAC_MIGRATION:
    enable: false
  POPULATE_VERSION_MIGRATION:
    enable: false
    record_update_limit: 100
  COLLABORATOR_RESOURCE_MIGRATION:
    enable: true
    record_update_limit: 100
```

but you have implemented it as list

Reply:
Yes, because order of migrations matter. By using a list, we enforce a specific order. A map can ran them in any order

Reply:
Ok then list is fine for but we need to add `enable: true` in list